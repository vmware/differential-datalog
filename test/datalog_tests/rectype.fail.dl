// recursive type definitions

typedef t = t

//---

typedef t = ( t, t )

//---

typedef t = t1
typedef t1 = (t, t)

//---

typedef f = undef

//---

// Type with no fields
typedef t = { }