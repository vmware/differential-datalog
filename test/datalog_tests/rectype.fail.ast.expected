typedef t = ttypedef t = (t, t)typedef t = t1
typedef t1 = (t, t)typedef f = undefFailed to parse input file: "./test/datalog_tests/rectype.fail.dl" (line 4, column 13):
unexpected "{"
expecting "bit", "int", "string", "bool", identifier or "("